environment.yml

```diff
@@ -4,13 +4,14 @@ channels:
     - bioconda
     - defaults
 dependencies:
-    - black=21.10b0
+    - black=22.10
```

Comment:
Black version `21.10b0` has having incompatibilities errors with click. If users have click version `>8.1`, black will crash with:

> ImportError: cannot import name '_unicodefun' from 'click'

To fix this, users will need to downgrade click down to `8.0`.

This problem has been fixed in Black `22.3` and up.

Reply:
It doesn't much matter which version of Black is used, as long as it's used consistently. So you're welcome to upgrade and pin a newer version that doesn't have these issues. But that's often best left to a dedicated thread, since it can result in numerous trivial formatting changes that add a lot of noise and clutter to an existing PR.